app/src/main/java/org/keynote/godtools/android/fragment/LanguagesFragment.java

```diff
@@ -98,14 +119,18 @@ void onLoadLanguages(@Nullable final List<Language> languages) {
         if (languages == null) {
             mLanguages = null;
         } else {
-            mLanguages = Stream.of(languages)
-                    .sorted((l1, l2) -> l1.getDisplayName().compareToIgnoreCase(l2.getDisplayName()))
-                    .toList();
+            mLanguages = sortLanguages(languages);
```

Comment:
this change is no longer necessary and can be reverted, the `sortLanguages` logic is only triggered here.